### PR TITLE
Small code improvements for TShellManager

### PR DESCRIPTION
**Pull request by abogutskiy**
Fix cosmetic issues for https://github.com/ytsaurus/ytsaurus/pull/719

### CODE REVIEW

**Review by gritukan — APPROVED** [review]

**Comment by maxim-babenko** [comment]
Shipped!

File: yt/yt/server/lib/shell/shell_manager.cpp [hunk 1]
```diff
@@ -169,20 +169,18 @@ class TShellManagerBase
         std::optional<int> shellIndex)
     {
         if (shellId) {
-            auto shell = Find(*shellId);
-            if (shell) {
+            if (auto shell = Find(*shellIndex); shell) {
```
Comment (maxim-babenko) [inline yt/yt/server/lib/shell/shell_manager.cpp:172]: ``; shell`` is redundant (here and below)

Comment (maxim-babenko) [reply yt/yt/server/lib/shell/shell_manager.cpp:172]: Also this is a wrong find:)

**Review by maxim-babenko — CHANGES_REQUESTED** [review]

**Comment by maxim-babenko** [comment]
Shipped!

**Pull request closed without merging**